pkg/pillar/types/dpc.go

```diff
@@ -750,6 +750,18 @@ type WirelessConfig struct {
 	Cellular []DeprecatedCellConfig
 }
 
+// IsEmpty returns true if the wireless config is empty.
+func (wc WirelessConfig) IsEmpty() bool {
```

Comment:
I believe it works as expected, but I'm trying to understand when this new check is necessary and when not... Why, for example, it's not necessary here https://github.com/lf-edge/eve/blob/f55efa1cf7332ff8739dfa1f15421d95e649c14a/pkg/pillar/dpcmanager/dns.go#L78 ?

Reply:
In that case it would be redundant because `wwanNetStatus` will be returned as nil if there is no corresponding config.